src/components/Steps/__tests__/steps-snapshot-tests.jest.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import renderer from "react-test-renderer";
```

Comment:
some of the tests check the "steps with numeric type" and some of them check "steps with gallery type" - please wrap each group of test with describe which explain to which type of steps each tests belong